service/src/test/java/bio/terra/cbas/initialization/TestCloneRecoveryServiceUnits.java

```diff
@@ -50,67 +49,15 @@ void generateTemplateUpdateManifest() {
     // only cloned run sets should be listed in the manifest.
     // the latest cloned run set should be the new template.
     // all other clone run sets should be listed as non-templates.
-    assertEquals(1, manifest.templateRunSets().size());
-    assertEquals(2, manifest.nonTemplateRunSets().size());
-    assertTrue(manifest.templateRunSets().contains(clonedRunSetLatest));
-    assertTrue(manifest.nonTemplateRunSets().contains(clonedTemplate));
-    assertTrue(manifest.nonTemplateRunSets().contains(clonedRunSet));
+    assertEquals(1, manifest.keepAsTemplate().size());
+    assertEquals(2, manifest.toBeDeleted().size());
+    assertTrue(manifest.keepAsTemplate().contains(clonedRunSetLatest));
```

Comment:
Could you test actual set membership instead of just contains?
eg `assertEquals(manifest.keepAsTemplate, List.of("clonedRunSetLatest"))`